### PR TITLE
Fix pyarrow build error on GDS server

### PR DESCRIPTION
**Pull request by brs96**

### CODE REVIEW

**Comment by brs96** [comment]
Combined with https://github.com/neo-technology/graph-analytics/pull/5983

**Review by adamnsch — APPROVED** [review]
:+1: 

I think we can still merge this as the error we see in CI seems totally unrelated

**Comment by brs96** [comment]
[[PF] Find out flaky python client](https://trello.com/c/A4nJM7qD/6614-pf-find-out-flaky-python-client)

File: graphdatascience/tests/integration/test_model_ops.py [hunk 1]
```diff
@@ -48,7 +48,7 @@ def lp_model(runner: Neo4jQueryRunner, gds: GraphDataScience, G: Graph) -> Gener
     try:
         pipe.addNodeProperty("degree", mutateProperty="rank")
         pipe.addFeature("l2", nodeProperties=["rank"])
-        pipe.configureSplit(trainFraction=0.4, testFraction=0.2, validationFolds=2)
+        pipe.configureSplit(trainFraction=0.7, testFraction=0.2, validationFolds=2)
```
Comment (Mats-SX) [inline graphdatascience/tests/integration/test_model_ops.py:51]: so weird that this only failed sometimes!

**Pull request merged**